web5-test-vectors/vector-structure.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "Describe the feature under test",
+    "numberOfTests": 3,
+    "vectors": [
+        {
+            "testId": 1,
```

Comment:
Would it make sense to use `description` as the `testId`? I worry that having both can easily become confusing.

Reply:
Agree might be annoying to keep these up to date if tests are removed/added. Might just leave the top level number, when consumers use, can check that they ran the expected number of vectors. And the description will indicate which tests succeeded/failed?

Reply:
+1 to using the description. I would still remove `numberOfTests`. Consumers should be able to easily tell if they've run against all vectors without having to check another field.